lib/ansible/plugins/lookup/mongodb.py

```diff
@@ -62,7 +62,9 @@
             type: integer
         sort:
             description:
-                - Sorting rules. Please notice the constats are replaced by strings.
+                - Sorting rules.
+                - Please use the strings C(ASCENDING) and C(DESCENDING) to set the order.
+                - Check the example for more information.
             type: list
             default: "[]"
```

Comment:
Plase also add extra_parameters. You are mentioning it in examples, but not in documentation. It could be little bit confusing
```suggestion
            default: "[]"
        extra_connection_parameters:
            description:
                - Extra connection parameters, eg. { "readPreference": "nearest", "ssl" : True , "ssl_certfile": "/etc/self_signed_certificate.pem" }
            type: 'dict'
            default: '{}'
```

Reply:
Once again, thanks for having noticed this. It's now added into the documentation as well.